code/modules/clothing/head/helmet.dm

```diff
@@ -17,6 +17,9 @@
 		"Drask" = 'icons/mob/species/drask/helmet.dmi'
 		)
 
+/obj/item/clothing/head/helmet/modern

```

Comment:
you really shouldn't make subtypes if you don't need to, there is a theoretical max on typepaths
just change the sprite